torchvision/transforms.py

```diff
@@ -873,15 +896,27 @@ def __init__(self, *args, **kwargs):
 
 
 class FiveCrop(object):
-    """Crop the given PIL Image into four corners and the central crop.abs
+    """Crop the given PIL Image into four corners and the central crop
 
-       Note: this transform returns a tuple of images and there may be a mismatch in the number of
-       inputs and targets your `Dataset` returns.
+    .. Note::
+         This transform returns a tuple of images and there may be a mismatch in the number of
+         inputs and targets your Dataset returns. See below for an example of how to deal with
+         this.
 
-       Args:
-           size (sequence or int): Desired output size of the crop. If size is an
-               int instead of sequence like (h, w), a square crop (size, size) is
-               made.
+    Args:
+         size (sequence or int): Desired output size of the crop. If size is an ``int``
+            instead of sequence like (h, w), a square crop of size (size, size) is made.
+
+    Example:
+         >>> transform = Compose([
+         >>>    FiveCrop(size), # this is a list of PIL Images
+         >>>    Lambda(lambda crops: torch.stack([ToTensor()(crop) for crop in crops])) # returns a 4D tensor
```

Comment:
nit: you don't need to use the module interface of `ToTensor` here, a `to_tensor(crop)` would work just fine. Maybe we should modify this in this case, what do you think?

Reply:
Yeah, I thought about this but went this way for the following reasons:
1) The docs for the functional interface aren't exposed (yet) so users don't know they even exist unless they check the codebase 
2) this is the docstring on the Object interface

I think we should fix 1 by probably re-arranging the transforms.py into a folder with an `__init__.py`. We may also want to split the functional interface into `transforms.functional` or something like that. 

Happy to change this to `to_tensor` (it's cleaner) if you think it's ok that this function isn't exposed in the docs yet